docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+services:
+  php:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
```

Comment:
I think we should have `.docker` directory instead. I've seen it somewhere and I think it's better convention.
In other repos we usually uses `.ci` dir for travis config.